jobserver/management/commands/map_existing_workspaces.py

```diff
@@ -0,0 +1,55 @@
+import csv
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from jobserver.authorization import ProjectCoordinator, ProjectDeveloper
+from jobserver.models import (
+    Org,
+    OrgMembership,
+    Project,
+    ProjectMembership,
+    User,
+    Workspace,
+)
+
+
+class Command(BaseCommand):
+    @transaction.atomic()
+    def handle(self, *args, **options):
+        with open("project-mappings.csv") as f:
```

Comment:
where does this come from?

Reply:
I've got it locally on my machine currently, including a command to generate it from the giant dict I've been using to collate all the information.  While the site and code is public I wasn't confortable making this block of data public in case it exposed a project which hadn't been publicised yet.